packages/l/libvpx/xmake.lua

```diff
@@ -0,0 +1,31 @@
+package("libvpx")
+    set_homepage("https://chromium.googlesource.com/webm/libvpx/")
+    set_description("libvpx is a free software video codec library from Google and the Alliance for Open Media (AOMedia)")
+    set_license("BSD-3-Clause")
+
+    add_urls("https://github.com/webmproject/libvpx/archive/refs/tags/v1.14.1.tar.gz")
+    add_versions("v1.14.1", "901747254d80a7937c933d03bd7c5d41e8e6c883e0665fadcb172542167c7977")
+
+    add_deps("autoconf", "automake", "libtool", "m4", "yasm", "nasm")
+
+    add_configs("vp8", {description = "enable the vp8 codec", default = false, type = "boolean"})
+    add_configs("vp9", {description = "enable the vp9 codec", default = false, type = "boolean"})
+    add_configs("vp9-post", {description = "vp9 specific postprocessing", default = false, type = "boolean"})
```

Comment:
rename to `vp9_post`, it will simply configuration. 

`add_requires("xx", {configs = {["vp9-post"] = true}})`
-> `add_requires("xx", {configs = {vp9_post = true}})`